move-vm-backend/tests/move_vm.rs

```diff
@@ -502,3 +509,39 @@ fn dry_run_gas_strategy_doesnt_update_storage() {
         "received module although the dry run strategy was enabled"
     );
 }
+
+#[test]
+// TODO(rqnsom): this one should not panic once the native functions are implemented
+#[should_panic]
+// This test heavily depends on Move.toml files for the used Move packages.
+fn genesis_config_initializes_substrate_stdlib() {
```

Comment:
The name is not correct - this doesn't have anything to do with `genesis_config`. I shall fix it.

Reply:
Done